src/safeds_stubgen/api_analyzer/_ast_visitor.py

```diff
@@ -263,10 +273,78 @@ def enter_funcdef(self, node: mp_nodes.FuncDef) -> None:
                 # Sort for the snapshot tests
                 type_var_types.sort(key=lambda x: x.name)
 
+        # Check docstring parameter types vs code parameter type hint
+        for i, argument in enumerate(arguments):
```

Comment:
```suggestion
        for i, parameter in enumerate(parameters):
```

Also rename the variable `arguments` to `parameters`. Arguments are at the call-site, paramters are at the declaration-site.